Geometry/HGCalGeometry/src/HGCalGeometryLoader.cc

```diff
@@ -114,6 +114,8 @@ HGCalGeometry* HGCalGeometryLoader::build (const HGCalTopology& topology) {
       }
     }
   }
+  // FIXME: Sort valid DetIds if neded
+  // geom->sortDetIds();
```

Comment:
@lgray - I'm a bit concerned about sorting 5 million element vector for "just in case" - as it's n*log2(n). What is current/future access pattern? Do you check every hit DetId on its validity? Mind the search is linear.